docs/whatsnew_1.5.7.html

```diff
@@ -150,10 +149,10 @@ <h1>
                 With that explained, it is important that whenever you upgrade your site, you should also inspect ALL the template_default files to determine which changes in those files need to be replicated in your customized files.
             </p>
             <p>
-                Often a tool like <a href="https://www.scootersoftware.com/download.php" target="_blank">Beyond Compare</a> or Araxis Merge or GnuDiff can be helpful for this.
+                Comparison/Merging Tools like <a href="https://www.scootersoftware.com/download.php" target="_blank">Beyond Compare</a> or Araxis Merge or GnuDiff are essential for this process.
```

Comment:
This change is good. 
Suggestion for further improvement: link to docs page and only mention tools we mention there.
 
Comparison/Merging Tools like <a href="https://www.scootersoftware.com/download.php" target="_blank">Beyond Compare</a> or WinMerge are essential for this process.  See 
 https://docs.zen-cart.com/user/first_steps/useful_tools/#file-comparison-utility